src/ucs/vfs/base/vfs_obj.c

```diff
@@ -46,6 +47,9 @@ struct ucs_vfs_node {
     ucs_list_link_t        list;
     void                   *arg_ptr;
     uint64_t               arg_u64;
+    ucs_vfs_node_t         *target;
+    ucs_list_link_t        links;
```

Comment:
1. can you please document the fields in this struct?
2. can we place some of them in a union now that there are so many? for example {text_cb, target} are probably exclusive